code/game/gamemodes/machinegod/machinegod_items.dm

```diff
@@ -0,0 +1,557 @@
+	T.turf_animation('icons/obj/clockwork/structures.dmi',"energyoverlay[pick(1,2)]",0,0,MOB_LAYER+1,'sound/weapons/bladeslice.ogg')
+	var/mob/living/M = A
+	if(!istype(M)) return
+	if(iscultist(M))
+		M.take_organ_damage(15)
+		M.Stun(1)
+	if(issilicon(M))
+		M.take_organ_damage(8)
+		M.Stun(2)
+	qdel(src)
+
+/spell/targeted/equip_item/clockspear
+	name = "Conjure Spear"
+	desc = "Conjure a brass spear that serves as a viable weapon against heathens and silicons. Lasts for 3 minutes."
+
+	spell_flags = 0
+	range = 0
+	charge_max = 2000
+	duration = 1800
+	invocation = "Rar'zl orjner!"
+	invocation_type = SpI_SHOUT
+	still_recharging_msg = "<span class='clockwork'>\"Patience is a virtue.\"</span>"
+	delete_old = 0
+
+	compatible_mobs = list(/mob/living/carbon/human)
+
+	override_base = "clock"
+	cast_sound = 'sound/effects/teleport.ogg'
+	hud_state = "clock_spear"
+
+/spell/targeted/equip_item/clockspear/New()
+	..()
+	equipped_summons = list("[slot_r_hand]" = /obj/item/weapon/spear/clockspear)
+
+/spell/targeted/equip_item/clockspear/choose_targets(mob/user = usr)
+	return list(user)
+
+/spell/targeted/equip_item/clockspear/cast(list/targets, mob/user = usr)
+	..()
+	playsound(user,'sound/effects/evolve.ogg',100,1)
+	for(var/turf/T in get_area(user))
+		for(var/obj/machinery/light/L in T.contents)
+			if(L && prob(20)) L.flicker()
+
+
+/obj/item/device/mmi/posibrain/soulvessel
+	name = "positronic brain"
+	desc = "A cube of ancient, glowing metal, three inches to a side and embedded with a cogwheel of sorts."
+	icon = 'icons/obj/assemblies.dmi'
+	icon_state = "soulvessel"
+	w_class = 2
+	origin_tech = "engineering=5;materials=5;bluespace=2;programming=5"
+
+	req_access = null
+
+/obj/item/device/mmi/posibrain/soulvessel/check_observer(var/mob/dead/observer/O)
+	if(jobban_isbanned(O, ROLE_CLOCKCULT))
+		return 0
+	if(..())
+		return 1
+	return 0
+
+//DEBUG ITEMS
+/obj/item/weapon/clockcheat
+	name = "ZOOP"
+	desc = "ZOP"
+	icon = 'icons/obj/assemblies.dmi'
+	icon_state = "soulvessel-occupied"
+
+/obj/item/weapon/clockcheat/attack_self(mob/living/user as mob)
+	if(!isclockcult(user))
+		user << "<span class='clockwork'>Ratvar: \"wyd?\"</span>"
+		return 0
+	user << "<span class='clockwork'>Ratvar: \"It's lit.\"</span>"
+	return 1
+
+/obj/item/weapon/clockcheat/spear
+	color = "#ff0000"
+
+/obj/item/weapon/clockcheat/spear/attack_self(mob/living/user as mob)
+	if(!..()) return
+	user.add_spell(new/spell/targeted/equip_item/clockspear)
+
+/*/obj/item/weapon/clockcheat/revenant
+	color = "#ff0000"
+
+/obj/item/weapon/clockcheat/revenant/attack_self(mob/living/user as mob)
+	if(!..()) return
+
+	var/mob/living/simple_animal/bound_revenant/R = new
+	R.contractor = user.mind*/
+
+/obj/item/weapon/clockcheat/gateway
+	color = "#ffff00"
+
+/obj/item/weapon/clockcheat/gateway/attack_self(mob/living/user as mob)
+	if(!..()) return
+
+	var/list/L = list()
+	var/list/areaindex = list()
+
+	for(var/obj/machinery/clockobelisk/C in world)
```

Comment:
just make a global list list `var/global/list/clockobelisks = list()` and then add any obelisks to that, then add any obelisks to that.

make sure to remove them in Destroy() though